docker/README.md

```diff
@@ -0,0 +1,79 @@
+## Running DC/OS UI in Docker
+
+These instructions will guide you through building and launching the DC/OS UI using docker.
+
+### Requirements
+1. [Install docker](https://www.docker.com/products/overview)
+2. Clone dcos-ui repo
+
+  ```sh
+  git clone git@github.com:dcos/dcos-ui.git && cd dcos-ui
+  ```
+
+### Build Step
+
+You'll only need to run the build once. If you change the path to your plugins repo, you'll need to build again.
+
+1. Start docker on your machine
+2. Build the dcos-ui docker image. Run this command inside the the cloned `dcos-ui` repo.
+
+  **Option 1**. Build without a plugins repo
+    ```sh
+    docker build \
+      -t dcos-ui-browse:latest \
+      -f docker/Dockerfile.browse .
+    ```
+
+  **Option 2**. Build with a plugins repo
+
+    If you have a repository of plugins to enhance DC/OS UI, you may configure this by passing a URL to the git repo. For example: `https://{AUTH_TOKEN}:x-oauth-basic@github.com/my-company/dcos-ui-plugins.git`, where `{AUTH_TOKEN}` would be your generated token from, e.g. https://github.com/settings/tokens.
+
+    ```sh
+    docker build \
+      --build-arg PLUGINS_REPO="" \
+      -t dcos-ui-browse:latest \
+      -f docker/Dockerfile.browse .
+    ```
+
+
+### Run step
+
+**Pre-requisite:** You must have an already running DC/OS cluster.
+
+1. Start docker on your machine
+
+2. Set cluster URL. The address of a DC/OS cluster to be used as the backend. The cluster version must be compatible with the UI assets.
+
+  ```sh
+  export CLUSTER_ADDRESS=https://a-dcos-cluster.us-west-2.elb.amazonaws.com
+  ```
+
+3. Run the dcos-ui docker image
+
+  **Option 1**. Run without a plugins repo
```

Comment:
Instead of having option 1, option 2, maybe we could have:
### Run the dcos-ui docker image
`docker run --ipc=host -it dcos-ui-browse`

**Options**: (configure options by adding `-e "option_name=option_value"` to command) e.g. to add a cluster address: `docker run --ipc=host -e "CLUSTER_ADDRESS=https://blah/blah.com" -it dcos-ui-browse`

**CLUSTER_ADDRESS**: (default: none - UI will use fixtures) https address of cluster.
**APPLICATION_CHECKOUT**: (default: master) Branch or commit hash to checkout
...